boofuzz/__init__.py

```diff
@@ -621,6 +622,27 @@ def s_byte(value, endian=LITTLE_ENDIAN, output_format="binary", signed=False, fu
     byte = primitives.Byte(value, endian, output_format, signed, full_range, fuzzable, name)
     blocks.CURRENT.push(byte)
 
+def s_bytes(value, size=-1, padding=six.binary_type(b"\x00"), fuzzable=True, max_len=-1, name=None):
+    """
+    Pushes a bytes field onto the current block stack.
```

Comment:
Great job! Maybe using `b""` directly, as discussed in other place.

> For future reference, we will use the standard language feature b'' type instead of six.binary_type. Python 2.5 compatibility isn't a goal for us. It's functionally fine either way though so this is fine.